src/Linkification/Embedding.js

```diff
@@ -592,30 +596,92 @@ var Embedding = {
+
+              function renderMedia(tweet) {
+                const mediaItems = tweet?.media?.all || [];
+                let media = ''
+                let photos = 1;
+                for (let i = 0; i < mediaItems.length; i++) {
+                  const mediaItem = mediaItems[i];
+                  const media_item_url = E(mediaItem.url);
+                  switch (mediaItem.type) {
+                    case 'photo':
+                      media += `<a target="_blank" href="${E(tweet.url)}/photo/${photos}"><img src="${media_item_url}" style="max-width: 80vw; max-height: 80vh;"></a>`
+                      photos += 1;
+                      break;
+                    case 'video':
+                    case 'gif':
+                      media += `<video controls="" preload="auto" src="${media_item_url}" style="max-width: 80vw; max-height: 80vh;"${(mediaItem.type == 'gif' ? ' loop' : '')}></video>`
+                      break;
+                    default:
+                      break;
+                  }
+              }
+              return media;
+            }
+
+            function renderDate(tweet) {
+              return new Date(tweet.created_at).toLocaleString(undefined, { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' });
+            }
+
+            function renderPoll(tweet) {
+              let poll = '<ul>'
+              tweet.poll.choices.forEach(choice => {
+                poll += `
+                    <li>${E(choice.label)} / ${E(choice.percentage)}%</li>
+                `
+              })
+              poll += `</ul>${+tweet.poll.total_votes || 0}&nbsp;votes &CenterDot; ${E(tweet.poll.time_left_en || '')}`
+              return poll;
+            }
+
+            const media = renderMedia(tweet);
+            let quote = ''
+            if (tweet?.quote) {
+              const quote_poll = (tweet?.quote?.poll) ? renderPoll(tweet.quote.poll) : ''
+              quote = `<hr/><blockquote>
+                <div style="display: flex;padding-bottom: 1em;">
+                  <a href="${E(tweet.quote.url)}">
+                  <div>
+                    <img src="${E(tweet.quote.author.avatar_url)}" style="width: 24px;transform: translateX(-50%) translateY(-50%);border-radius: 9999px;">
+                  </div>
+                  <div style="margin: -2.25em 0 0 1em;">${E(tweet.quote.author.name)} (@${E(tweet.quote.author.screen_name)}) ${renderDate(tweet.quote)}</div>
+                  </a>
+                </div>
+                <p lang="${tweet.quote?.lang || 'en'}" dir="ltr" style="margin-top: 0">${E(tweet.quote.text)}</p>
+                ${renderMedia(tweet.quote)}
+                ${quote_poll}
+              </blockquote>`
+            }
+
+            const poll = (tweet?.poll) ? renderPoll(tweet) : '';
+            const created_at = renderDate(tweet);
+
+            const innerHTML = `
+              <p lang="${tweet.lang}" dir="ltr">${E(tweet.text)}</p>
```

Comment:
Another unescaped variable. Maybe use the jsx function? That escapes automatically.